src/Perlang.Interpreter/Typing/TypesResolvedValidator.cs

```diff
@@ -355,28 +378,19 @@ public override VoidObject VisitVarStmt(Stmt.Var stmt)
                     }
                 }
             }
-            else if (stmt.Initializer == null)
+            else if (!stmt.TypeReference.ExplicitTypeSpecified && stmt.Initializer == null)
             {
                 TypeValidationErrorCallback(new TypeValidationError(
-                    null,
+                    stmt.Name,
```

Comment:
This is one of the problems referred to in this comment: https://github.com/perlang-org/perlang/pull/279/files#r809233300. The previous way this was written would cause runtime errors in the `Perlang.ConsoleApp.Program.Error` method. Particularly nasty, since _these errors are typically not provoked by tests_. Needless to say, this is just one example of why doing #39 sooner than later would be a Really Good Thing.&trade;